client-library/src/honoMiddleware.ts

```diff
@@ -94,7 +96,15 @@ export function createHonoMiddleware<App extends HonoApp>(
 
     const service = env<FpxEnv>(c).FPX_SERVICE_NAME || "unknown";
 
-    const ctx = c.executionCtx;
+    const ctx =
+      getRuntimeKey() === "workerd"
+        ? c.executionCtx
+        : {
+            // HACK - Untested
+            waitUntil: async (p: Promise<unknown>) => {
```

Comment:
NOTE - Should swallow this error somehow, otherwise it will be unhandled and crash the Deno runtime